src/data/real/ennreal.lean

```diff
@@ -500,6 +500,20 @@ coe_mono.map_min
 @[norm_cast] lemma coe_max : ((max r p:ℝ≥0):ennreal) = max r p :=
 coe_mono.map_max
 
+lemma le_of_to_nnreal_le_of_top_imp_top {a b : ennreal} (h : a = ⊤ → b = ⊤)
```

Comment:
The order in the name should match the order of assumptions, so you should either change the name or the order of assumptions.

Reply:
I changed the name.